xgcm/grid.py

```diff
@@ -1200,18 +1226,23 @@ def __init__(
             for key, value in metrics.items():
                 self.set_metrics(key, value)
 
-    def _parse_axes_kwargs(self, kwargs):
-        """Convvert kwarg input into dict for each available axis
+        # Finish setup
+
+    def _as_axis_kwarg_mapping(self, kwargs, axes=None):
+        """Convert kwarg input into dict for each available axis
```

Comment:
Another place where type hinting would be very nice

Reply:
Getting right on it...

Reply:
does `self` get a type hint?

Reply:
I added type hints, but am very unsure if I did it correctly. Would you mind checking?

Reply:
`self` doesn't need one (unless your method returns self later, which yours doesn't).

Your type hints look okay to me! Just need to add a return value to this function.